### PR TITLE
Oven pr

### PR DESCRIPTION
**Pull request by snorkyware**

### CODE REVIEW

File: ValheimPlus/GameClasses/CookingStation.cs [hunk 1]
```diff
@@ -89,4 +90,66 @@ private static ItemDrop.ItemData PullCookableItemFromNearbyChests(CookingStation
...
+        }
+
+        [HarmonyPatch(typeof(CookingStation), nameof(CookingStation.UpdateFuel))]
+        public static class CookingStation_UpdateFuel_Patch
+        {
+            // If stay-at-max-fuel mode, fuel is set to max in Awake,
+            // so simply cancel this call so that fuel level is never decremented.
+            private static bool Prefix(CookingStation __instance) =>
+                !__instance.m_useFuel || !Configuration.Current.Oven.IsEnabled ||
+                !Configuration.Current.Oven.infiniteFuel;
```
Comment (Grantapher) [inline ValheimPlus/GameClasses/CookingStation.cs:116]: I'm not allowing method skipping unless it is absolutely necessary. This should be able to be done without method skipping, can you update it to do so?

**Review by Grantapher — CHANGES_REQUESTED** [review]
Just one main comment.

Also, when this is force-pushed, can you squash the commits?

File: ValheimPlus/GameClasses/CookingStation.cs (hunk 1 above)
Comment (snorkyware) [reply ValheimPlus/GameClasses/CookingStation.cs:116]: Okay, this is good advice. I'll have a look if I get time. Please feel free to make whatever changes you see fit in the mean time.

**Comment by Grantapher** [comment]
Committed in 0978253.

**Pull request closed without merging**